modules/ocf_printhost/files/enforcer

```diff
@@ -103,13 +103,26 @@ ENFORCER_ERROR_MESSAGE = Message(
         """) + MAIL_SIGNATURE
 )
 
+NOTIFY_QUOTA_MESSAGE = dedent("""\
+        Your print job failed due to insufficient pages. Your job was
+        {pages} pages, and you have {quota} pages remaining today.""")
```

Comment:
I'd put the ending parentheses on the next line, since otherwise the indentation looks like it doesn't have an end. Like this:

```
NOTIFY_QUOTA_MESSAGE = dedent("""\
        Your print job failed due to insufficient pages. Your job was
        {pages} pages, and you have {quota} pages remaining today."""
)
```

Same with the other two messages